service-front/module/Application/config/module.config.php

```diff
@@ -502,5 +522,39 @@
             'po_ind' => 'International driving licence (must be current)',
             'po_n' => 'None of the above',
         ],
+        'non_uk_identity_methods' => [
+            'xpn' => 'Passport',
+            'xdln' => 'Photocard driving licence',
+            'xid' => 'National identity card',
+        ],
+        'acceptable_nations_for_id_documents' => [
+            'Austria',
+            'Belgium',
+            'Bulgaria',
+            'Croatia',
+            'Republic of Cyprus',
+            'Czech Republic',
+            'Denmark',
+            'Estonia',
+            'Finland',
+            'France',
+            'Germany',
+            'Greece',
+            'Hungary',
+            'Ireland',
+            'Italy',
+            'Latvia',
+            'Lithuania',
+            'Luxembourg',
+            'Malta',
+            'Netherlands',
+            'Poland',
+            'Portugal',
+            'Romania',
+            'Slovakia',
+            'Slovenia',
+            'Spain',
+            'Sweden',
```

Comment:
These should be identified and saved as ISO 3166 country codes, then translated into names in the UI. It makes our storage more canonical, and is necessary to pass them to the Yoti API (which uses 3-char ISO-3166).

You should also check these names against [the FCDO list](https://assets.publishing.service.gov.uk/media/65fd8475f1d3a0001132adf4/FCDO_Geographical_Names_Index_March_2024.csv/preview). For example "Republic of Cyprus" should be "Cyprus" and "Czech Republic" should be "Czechia".